library/std/src/panicking.rs

```diff
@@ -277,7 +278,7 @@ fn default_hook(info: &PanicHookInfo<'_>) {
 
             let write_msg = |dst: &mut dyn crate::io::Write| {
                 // We add a newline to ensure the panic message appears at the start of a line.
-                writeln!(dst, "\nthread '{name}' panicked at {location}:\n{msg}")
+                writeln!(dst, "\nthread '{name}' ({tid:#x}) panicked at {location}:\n{msg}")
```

Comment:
Currently this prints as hex because I am accounting for some platforms providing a u64 or address ID (Apple) or Rust's `ThreadId` (u64 repr). But it seems like the Apple TIDs still typically fit within a `u32` and Rust TIDs are currently sequential, so I plan to change it back to a decimal (which matches `ps` and debugger output).

Reply:
Did this change.